roles/openshift/tasks/firewalld.yml

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
The other benefit to the single role is that it makes it easier to preserve ordering without requiring the user to know which order to include specific roles. I've found the more simplistic we can make our tooling for our users, the better the experience is for them.